<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.5, y2:0.5, stop:0 rgba(50, 50, 50, 200), stop:1 rgba(190, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="LabelRecorte1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Volver">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelRecorte2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetSeleccion">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Resetear selecci√≥n</string>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Primer recorte</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Segundo recorte</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>296</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 13pt &quot;Helvetica&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Ingresar datos de paciente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string>Edad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Sexo (F/M):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>Puntaje de examen mini-mental:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CargarDatos">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 10pt &quot;Helvetica&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Avanzar">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold</string>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Age">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Sex">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="MiniMentalScore">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Ayuda">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AyudaRecortes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
